react/features/base/dialog/components/web/StatelessDialog.js

```diff
@@ -29,6 +29,11 @@ const OK_BUTTON_ID = 'modal-dialog-ok-button';
 type Props = {
     ...DialogProps,
 
+    /**
+     * True if listening for the Enter key should be disabled.
+     */
+    disableEnter: boolean,
```

Comment:
why is this needed? if there no cancel button nor a submit one, nothing would happen, right? Maybe we could not add the listener in that case and make it implicit?

Reply:
I suppose this was the problem I reported, that when you hit enter on the join screen and it goes to the close page instead of submitting the form.

Reply:
I tried many combos, and ended up here

Reply:
What are you trying to solve with it?

Reply:
To avoid dismissing the dialog on enter. But the dialog must still be dismiss-able, just not using an enter.